build_scripts/build_usd.py

```diff
@@ -393,7 +393,10 @@ def RunCMake(context, force, extraArgs = None):
 
     # Note - don't want to add -A (architecture flag) if generator is, ie, Ninja
     if IsVisualStudio2019OrGreater() and "Visual Studio" in generator:
-        generator = generator + " -A x64"
+        if "ARM" in os.environ.get('PROCESSOR_IDENTIFIER'):
+            generator = generator + " -A arm64"
```

Comment:
I'm not super familiar with how its configured, but you might also need to support ARM64EC on Windows too ? https://learn.microsoft.com/en-us/windows/arm/arm64ec